pandas/io/formats/style.py

```diff
@@ -566,6 +567,7 @@ def to_excel(
         inf_rep: str = "inf",
         verbose: bool = True,
         freeze_panes: tuple[int, int] | None = None,
+        storage_options: StorageOptions = None,
```

Comment:
would need a versionadded 1.5

Reply:
added

Reply:
@jreback - we've been reporting storage_options as being part of this method in the docs since 1.2. I think the easiest way to make this "1.5" here instead is to add an argument to the NDFrame.to_excel docstring (e.g. `storage_options_versionadded` or something) and set it for all uses of this docstring. Do you think that's worth updating, or is it okay as is?

Reply:
@jreback - friendly ping